met-cs634-spring2020-team-b/apex-agility
# Feature / add register page

### Summary of Changes:
- add Get Inspired section to home page
- add Register page `/register`
- add `<BannerImage>` component
### QA/CR Instructions:
- view Get Inspired carousel on Home Page
- click 'become a trainer' in footer to view Register page
- view Assessment, Courses, and About Us pages to see new banner implemented without impacting the existing UI
